setup.py

```diff
@@ -222,9 +281,13 @@ class webhost(distutils.cmd.Command):
             'A path to the directory where Azure Web Host will be installed.'),
     ]
 
-    def initialize_options(self):
-        self.webhost_version = None
+    def __init__(self, dist: Distribution):
+        super().__init__(dist)
         self.webhost_dir = None
+        self.webhost_version = None
+
+    def initialize_options(self):
```

Comment:
Can this be removed?

Reply:
![image](https://user-images.githubusercontent.com/1047040/152445342-46de9d58-cfaf-421e-a11a-eb8953d7327a.png)

This is to implement the methods in abstract class Command.